clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AnnotationsBasedAtomicOperationsRegistry.groovy

```diff
@@ -35,6 +36,9 @@ class AnnotationsBasedAtomicOperationsRegistry extends ApplicationContextAtomicO
   @Autowired
   List<CloudProvider> cloudProviders
 
+  @Autowired(required = false)
```

Comment:
You might not need this, I think you'll just get an empty list if there aren't any `GlobalDescriptionValidator`s